Issue: IP requirements are unclear

sallain opened the pull request:
This clarifies the IP requirements for post-install configuration (centos, ubuntu,
and ansible), changing "public IP" to the more precise "an IP address or fully-
qualified domain name instead of local host."

Resolves #129.

sallain committed:
Issue: IP requirements are unclear

This clarifies the IP requirements for post-install configuration (centos, ubuntu,
and ansible), changing "public IP" to the more precise "an IP address or fully-
qualified domain name instead of local host."

Resolves #129.

sevein commented:
CC @hakamine @mamedin @scollazo

mamedin commented on admin-manual/installation-setup/installation/install-ubuntu.rst:
```diff
@@ -388,9 +388,9 @@ the configuration of your new server.
-   firewall rules and allow access only to ports 80 and 8000 for Archivematica
-   usage.
+   If you are using an IP address or fully-qualified domain name instead of
+   localhost, you will need to configure your firewall rules and allow access
```
I think it is a typo, it should be: ports 80 and 8000

sallain replied:
Good catch! Resolved.

sallain committed:
Remove "required" from ArchivesSpace Use Statement

In the ArchivesSpace administration setup, Use Statement is not
a required field, as was erroneously stated in the docs.

Resolves #106

mamedin commented on admin-manual/installation-setup/upgrading/upgrading.rst:
```diff
@@ -273,53 +212,71 @@ Upgrade from Archivematica |previous_version| for CentOS/Red Hat
+
+      sudo -u archivematica bash -c " \
+          set -a -e -x
+          source /etc/sysconfig/archivematica-dashboard
```
The`/etc/sysconfig/archivematica-dashboard` config file is only for CentOS. The Ubuntu's file is `/etc/default/archivematica-dashboard`. I think this should be indicated in a note or explicitly for each linux distribution.

sallain replied:
@mamedin can you start a new issue?

mamedin replied:
@sallain done: https://github.com/artefactual/archivematica-docs/issues/139

mamedin commented on admin-manual/installation-setup/upgrading/upgrading.rst:
```diff
@@ -273,53 +212,71 @@ Upgrade from Archivematica |previous_version| for CentOS/Red Hat
-       am rebuild-transfer-backlog /path/to/transfers
+      sudo -u archivematica bash -c " \
+          set -a -e -x
+          source /etc/sysconfig/archivematica-dashboard
```
The/etc/sysconfig/archivematica-dashboard config file is only for CentOS. The Ubuntu's file is /etc/default/archivematica-dashboard. I think this should be indicated in a note or explicitly for each linux distribution.
